jarbas_hive_mind/message.py

```diff
@@ -0,0 +1,143 @@
+from enum import Enum
+import json
+from ovos_utils.json_helper import merge_dict
+from mycroft_bus_client import Message
+
+
+class HiveMessageType(str, Enum):
+    HANDSHAKE = "shake"  # negotiate initial connection
+    BUS = "bus"  # request meant for internal mycroft-bus in master
+    SHARED_BUS = "shared_bus"  # passive sharing of message
+    # from mycroft-bus in slave
+    BROADCAST = "broadcast"  # forward message to all slaves
+    PROPAGATE = "propagate"  # forward message to all slaves and masters
+    ESCALATE = "escalate"  # forward message up the authority chain to all
+    # masters
+    HELLO = "hello"  # like escalate, used to announce the device
+    QUERY = "query"  # like escalate, but stops once one of the nodes can
+    # send a response
+    CASCADE = "cascade"  # like propagate, but expects a response back from
+    # all nodes in the hive (responses optional)
+    PING = "ping"  # like cascade, but used to map the network
+    RENDEZVOUS = "rendezvous"  # reserved for rendezvous-nodes
+
+    THIRDPRTY = "3rdparty"  # user land message, do whatever you want
```

Comment:
I think this becomes unnecessary with the introduction of Actions, plus the semi-analogous FREE codes. I should probably diff the current state of the binary proposal. It's got most of these, plus some others (the opcode side is an IntEnum, but they should share names)

Reply:
this is essentially a "ignore" message, allows any downstream to use this as a dumb transport layer and completely disregard the Actions/Registry. Either way i want this opcode in there, but im not sold on the current naming as it's not immediately clear what it is for.

what this means is "ignore this payload, just pass it along", then someone can decide to actually react to it but the protocol does not care